app/models/project.rb

```diff
@@ -60,6 +60,7 @@ def as_json(_options = nil)
     {
       block: block,
       ciProvider: ci_provider&.titleize,
+      criticality: criticality,
```

Comment:
Since the forms only support certain values for the new `criticality` field, it would make sense to also add a model validation enforcing that to this class. Something like:

```ruby
validates :criticality, inclusion: { in: [0, 1, 2, 3] }
```